src/app/_components/CreateUpdateProfileForm/CreateUpdateProfileForm.tsx

```diff
@@ -0,0 +1,92 @@
+import { useState } from "react";
+import { Box, TextField } from "@mui/material";
+import { Button } from "../Button";
+
+interface CreateUpdateProfileFormProps {
+  onCancel: () => void;
+  onSave: () => void;
+}
+
+const CreateUpdateProfileForm = ({
+  onCancel,
+  onSave,
+}: CreateUpdateProfileFormProps): JSX.Element => {
+  const [name, setName] = useState("");
+  const [username, setUsername] = useState("");
+  const [bio, setBio] = useState("");
+
+  const handleCancel = () => {
+    // Handle cancel action
+    console.log("Action Canceled");
```

Comment:
Should run these logs on the story and keep the console logs out of the actual component incase we forget to get rid of them later. Other then that looks good.

Reply:
Removed console logs inside the components and moved it into stories